src/API/Trace/Propagation/B3Propagator.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\API\Trace\Propagation;
+
+use OpenTelemetry\Context\Propagation\TextMapPropagatorInterface;
+
+/**
+ * B3 is a propagator that supports the specification for the header
+ * "b3" used for trace context propagation across service boundaries.
+ * (https://github.com/openzipkin/b3-propagation)
+ */
+final class B3Propagator
+{
+    public static function getB3SingleHeaderInstance(): TextMapPropagatorInterface
+    {
+        return B3SinglePropagator::getInstance();
```

Comment:
`::getB3SingleHeaderInstance()` and `::getB3MultiHeaderInstance()` have to return a `B3Propagator` that uses both propagators for extracting, similar to `::getInstance()` before the last commit.
```php
private function __construct(TextMapPropagatorInterface $propagator)
{
	$this->propagator = propagator;
}
public static function getB3SingleHeaderInstance(): self
{
	static $instance;
	return $instance ??= new self(B3SinglePropagator::getInstance());
}
public static function getB3MultiHeaderInstance(): self
{
	static $instance;
	return $instance ??= new self(B3MultiPropagator::getInstance());
}
```

Reply:
Implemented.